### PR TITLE
docs: Add steps for database backup/restoration

### PR DESCRIPTION
**Pull request by fw-noel**
Ticket: [HCAP-1356](https://eydscanada.atlassian.net/browse/HCAP-1351)

This adds some documentation around restoring backups and what to do when it all goes terribly wrong. It might not be obvious what some things are without looking at Openshift, so feel free to ping me if you don't have Openshift access yet

### CODE REVIEW

**Review by vanics — DISMISSED** [review]

File: docs/database.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ Backups are enabled by means of the [backup-container](https://github.com/BCDevO
 
 For an in-depth explanation of the backup-container project, see the [project README](https://github.com/BCDevOps/backup-container/blob/master/README.md). In short, backups will be persisted to off-site storage as well as validated. Validation is accomplished by restoring the database backup to a temporary database and running one or more trivial queries against the DB.
 
+As noted in the above README, backups are on a schedule while restoration must be done manually. The steps to restore are:
+- Scale down all resources that connect to the database - at this point, it is just the hcap-server DC pods
+- Ensure that cron jobs will not run while performing the restoration
+- Scale down patroni to a single node, ensuring that it is the leader
+-- To see which members are the leaders vs the replicas, while RSH-ing into the Patroni node, use `patronictl -c patroni.yml list`
```
Comment (rhattie-ey) [inline docs/database.md:27]: Are these supposed to be indented? If so, you'll want to add 4 spaces before the line, not an extra dash. Like so:

```
- Item 1
    - Item 1.1
- Item 2
```
Which becomes:

- Item 1
    - Item 1.1
- Item 2

Comment (fw-noel) [reply docs/database.md]: Oh yes, sorry, I meant to fix that before putting this up. Fixed.

**Review by rhattie-ey — APPROVED** [review]
Everything looks good to me. Approved.

**Comment by mmumby** [comment]
Also looks good to me! (Although My OS experience is level 1 basic)

**Review by mmumby — APPROVED** [review]

**Pull request merged**